docs/release_guide.md

```diff
@@ -121,3 +123,28 @@ Make an announcement on the [mailing list](https://groups.google.com/g/openservi
 Skip this step if the release is a release candidate (RC).
 
 Open a pull request against the `main` branch making the same version updates as [above](#update-release-branch-with-patches-and-versioning-changes) so the latest release assets are referenced there.
+
+## Make version changes on docs.openservicemesh.io
+
+Associated version numbers need to be updated in the [docs.openservicemesh.io](https://github.com/openservicemesh/osm-docs/) repo.
+
+The files that need to be updated are:
+
+* [Readme](https://github.com/openservicemesh/osm-docs/blob/main/README.md)
```

Comment:
What exactly needs to be updated here? I only see references to release branches as an example, but those don't need to stay up to date.